tools/rna_tools/rnaz/AnnotateRNAz.py

```diff
@@ -0,0 +1,206 @@
+# AnnotateRnaz.py ---
+#
+# Filename: AnnotateRnaz.py
+# Description:
+# Author: Joerg Fallmann
+# Maintainer:
+# Created: Sat Jan 26 12:45:25 2019 (+0100)
+# Version:
+# Package-Requires: ()
+# Last-Updated: Sat Jan 26 16:21:54 2019 (+0100)
+#           By: Joerg Fallmann
+#     Update #: 185
+# URL:
+# Doc URL:
+# Keywords:
+# Compatibility:
+#
+#
+
+# Commentary:
+# This script is a replacement for rnazAnnotate.pl
+# rnazAnnotate can not handle the output from version 2 adequatly
+# This script uses the bedtools API to fast intersect an annotation Bed
+# with output from RNAz
+
+# Change Log:
+#
+#
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+# Code:
+
+#!/usr/bin/env python3
+
+import sys
+import glob
+import argparse
+from io import StringIO
+import gzip
+import traceback as tb
+from pybedtools import *
+import pybedtools
```

Comment:
isn't this redundant?

Reply:
you could also order the imports lexicographically

Reply:
Hi @bernt-matthias I took this from the pybed example, but yes imho you are